### PR TITLE
Add description and documentation to README

### LINKED ISSUE #18: Complete README, please
Hello!

I want to know more about this gem.

You've already released version 1.0.0 of this gem.

Please, write description and usage.

Thank you.

### PR DESCRIPTION
**Pull request by robotbill**
Adds a short description and documentation of Email Repair's public
methods to the README.

resolves #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +23,72 @@ Or install it yourself as:
...
+* replaces commas with periods
+
+#### CommonDomainSuffixRepair
+
+Adds missing top-level domain for common email domains. For example, it replaces
+`blah@gmail` with `blah@gmail.com`.
+
+#### CommonDomainPeriodAdder
+
+Adds missing period between the domain and top-level domain for  common email
```
Comment (tylercallahan) [inline README.md:74]:
```suggestion
Adds missing period between the domain and top-level domain for common email
```

File: README.md [hunk 2]
```diff
@@ -22,7 +23,72 @@ Or install it yourself as:
...
+
+### Available Repairs
+
+#### CommonMistakeRepair
+
+Repairs common email typos:
+* downcases the email
+* removes white space
+* removes duplicate @ symbols
+* replaces `.c0m` with `.com`
```
Comment (tylercallahan) [inline README.md:64]: Ha, is this really common?!

File: README.md [hunk 3]
```diff
@@ -4,7 +4,8 @@
 [![Build Status](https://travis-ci.org/ChalkSchools/email-repair.svg?branch=master)](https://travis-ci.org/ChalkSchools/email-repair)
 [![Coverage Status](https://img.shields.io/coveralls/ChalkSchools/email-repair.svg)](https://coveralls.io/r/ChalkSchools/email-repair?branch=master)
 
-TODO: Write a gem description
+Email Repair is a utility for sanitizing and validating user provided email
```
Comment (tylercallahan) [inline README.md:7]:
```suggestion
Email Repair is a utility for sanitizing and validating user-provided email
```

File: README.md [hunk 4]
```diff
@@ -22,7 +23,72 @@ Or install it yourself as:
...
+#### EmailRepair::Mechanic#repair_all(emails)
+
+Takes an array of emails, applies available repairs to each email and returns an
+`OpenStruct` containing keys `sanitized_emails:` with a unique array of the
+emails that were able to be repaired, and `invalid_emails:` with unique array of
+the emails that were unable to be repaired
+
+#### EmailRepair::Constants::email_regex
+
+Returns a regular expression to be used to validate user supplied email
```
Comment (tylercallahan) [inline README.md:53]:
```suggestion
Returns a regular expression to be used to validate user-supplied email
```

File: README.md (hunk 2 above)
Comment (robotbill) [reply README.md:64]: Email Repair says it is!

**Pull request merged**